README.md

```diff
@@ -33,6 +34,12 @@ You can run this validator using a GTFS dataset on your computer, or from a URL.
 
 More detailed instructions with all the parameters that exists are available on our ["Usage"](/docs/USAGE.md) page.
 
+### <a name="visualize-results"></a>Visualize the results
```

Comment:
Nit - You don't need the anchor here, you can just link to `#visualize-the-results` below

Reply:
Interesting! I thought it wouldn't work because we have two "Visualize the results" titles, but the first one is properly referenced. We don't need it in this case, but do you know a way to reference the second one using the same method?